pkg/fdbstatus/status_checks.go

```diff
@@ -29,6 +29,23 @@ import (
 	"math"
 )
 
+// forbiddenStatusMessages represents messages that could be part of the machine-readable status. Those messages can represent
+// different error cases that have occurred when fetching the machine-readable status. A list of possible messages can be
+// found here: https://github.com/apple/foundationdb/blob/main/documentation/sphinx/source/mr-status.rst?plain=1#L68-L97
+// and here: https://apple.github.io/foundationdb/mr-status.html#message-components.
+// We don't want to block the exclusion check for all messages, as some messages also indicate client issues or issues
+// with a specific transaction priority.
+var forbiddenStatusMessages = map[string]fdbv1beta2.None{
+	"storage_servers_error":              {},
+	"status_incomplete":                  {},
+	"unreachable_master_worker":          {},
+	"unreachable_dataDistributor_worker": {},
+	"unreachable_ratekeeper_worker":      {},
+	"unreadable_configuration":           {},
+	"unreachable_processes":              {},
+	"log_servers_error":                  {},
+}
```

Comment:
thanks, discussed with @jzhou77 , we think that the above 4 are required, while `status_incomplete` and `unreachable_processes` are overkills. 

For `status_incomplete`, it is present whenever there are some status fields not giving full results, it could be some trace events not being read, or even it could be some features only applicable to Snowflake and FDB never returns them.

For `unreachable_processes`, it is present this way: status actor tries to collect multiple trace events from ALL workers, including `MachineMetrics`, `ProcessMetrics`, `NetworkMetrics` , `ProgramStart`, etc. If any of them fails to reply due to timeout or other reasons, `unreachable_processes` will be present. 

As a result, if we include these 2 guy here, I am not sure if we can ever proceed to exclude.

```
unreadable_configuration
full_replication_timeout
storage_servers_error
log_servers_error
```

Reply:
Thanks, for the feedback!